e2e/test/pod-pvc-readonly/create-pvc-pod.yaml

```diff
@@ -0,0 +1,51 @@
+allowVolumeExpansion: true
+apiVersion: storage.k8s.io/v1
+kind: StorageClass
+metadata:
+  name: (join('-', ['linode-block-storage', $namespace]))
+provisioner: linodebs.csi.linode.com
+reclaimPolicy: Delete
+volumeBindingMode: Immediate
+parameters:
+    linodebs.csi.linode.com/volumeTags: (to_string($namespace))
+---
+apiVersion: v1
+kind: PersistentVolumeClaim
+metadata:
+  name: pvc-filesystem
+spec:
+  accessModes:
+  - ReadWriteOnce
```

Comment:
Should this be ReadyOnly?

Reply:
So for access modes in PVC, we just support ReadWriteOnce. Since we are implementing the readonly on the Node side, we have to set readOnly flag in the Pod spec under volumeMount. This will ensure that flag gets passed to NodePublishVolumeRequest obj which is used by the CSI driver/Node server to mount it as read only!

Reply:
Ah ok, looks good to me then.